cinch/roles/jenkins_master/templates/etc/nginx/conf.d/jenkins_http.conf

```diff
@@ -0,0 +1,46 @@
+server_names_hash_bucket_size 6400;
+upstream jenkins {
+  server 127.0.0.1:8080 fail_timeout=0;
+}
+
+server {
+  listen 80; 
+  server_name {{ inventory_hostname }}; 
```

Comment:
Here, I disagree. I can't think about any better way of getting the host's hostname/fqdn. And if you're managing your infrastructure correctly, one would assume that you also use correct hostnames to connect to your hosts via Ansible (or anything else for that matter) - whether it's a real hostname or just an IP address. I currently don't see a better way to get this value considering we need either a valid DNS hostname or an IP address as a fallback.